### ISSUE TITLE
Will new version be released for macOS Big Sur

### ISSUE THREAD

**Issue by GHubbler** [robertklep/quotefixformac#96]
macOS Big Sur is out. So I'd like to ask, if there's any hope or plans to support it.

**Comment by robertklep** [comment]
I was actually considering installing Big Sur, after having skipped Catalina, but I keep reading about BS bricking Late 2013 MBP's, which I happen to have. So not feeling very confident about upgrading right now 😬

**Comment by macjeff** [comment]
I would love to have this for Big Sur too.  If you are not going to update I will trash the plugin but for now the old version is waiting for an update!!!

**Comment by herbs** [comment]
An update to Big Sur would be highly appreciated at this end.

**Comment by Rogue-Git-Dev** [comment]
I concur with all parties above. This plugin deserves a proper update for Big Sur.

**Comment by LwsBtlr** [comment]
The WWDC for 2021 features a session on the new MailKit framework. Based on what I see in there, Bundles are going away and MailKit will be the new way, and at this point it does not appear it allows modifying the compose message, only the compose window.

**Comment by dcorbe** [comment]
> at this point it does not appear it allows modifying the compose message, only the compose window.

https://developer.apple.com/documentation/mailkit/memessage

**Comment by dsedivec** [comment]
> > at this point it does not appear it allows modifying the compose message, only the compose window.
> 
> https://developer.apple.com/documentation/mailkit/memessage

Is this the first time Apple has properly documented any of the Mail.app "plug-in" API?  This seems cool.

I think the only opportunity for getting at the raw message body might be `MEComposeSession.mailMessage.rawData`, which purports to be the raw source of the message, headers and body.  I have a bad *feeling* that, in the case of composing a message (as opposed to reading a message), this will be nil.  If it's not then I wonder if you'd be allowed to modify it, and if you did, would that modification be properly applied to the compose window?  I'm very skeptical.

Reading through the docs, "modify the message body of the message being composed" doesn't sound like one of their use cases.

(Disclaimer: Not really an Objective-C programmer and I probably know even less about Swift)

**Comment by dcorbe** [comment]
Yeah I don't quite understand how that works and which handlers fire when the raw message data is updated.
